templates/gpg2_gen_key.j2

```diff
@@ -5,6 +5,8 @@ Name-Real: {{ aptly_company_name }}
 Name-Comment: aptly key no passphrase
 Name-Email: {{ aptly_key_email }}
 Expire-Date: 0
+%no-ask-passphrase
+%no-protection
```

Comment:
talk me through what no-protection does and the reason for adding it please?

Reply:
I found I needed this to create gpg keys with no password. The GPG [documentation](https://www.gnupg.org/documentation/manuals/gnupg/Unattended-GPG-key-generation.html) states: "Using this option allows the creation of keys without any passphrase protection."